specs/experimental/custom-gas-token.md

```diff
@@ -234,6 +252,8 @@ The following methods MUST revert when `CALLVALUE` is non zero:
 - `relayMessage(uint256,address,address,uint256,uint256,bytes)`
```

Comment:
We should remove `relayMessage` from this list and instead follow up saying that it should be impossible to call with non zero callvalue with custom gas token due to the enforcement in `sendMessage`, framing it as an implicit invariant is better than explicit since we don't explicitly enforce it like @mds1 suggested